toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneView.kt

```diff
@@ -102,26 +122,69 @@ fun TableTopSceneView(
     onDrawStatusChanged: ((DrawStatus) -> Unit)? = null,
     content: (@Composable TableTopSceneViewScope.() -> Unit)? = null
 ) {
+    var initializationStatus: TableTopSceneViewStatus by remember {
+        mutableStateOf(
+            TableTopSceneViewStatus.Initializing
+        )
+    }
+    val updateStatus = remember {
+        { newStatus: TableTopSceneViewStatus ->
+            initializationStatus = newStatus
+            onInitializationStatusChanged?.invoke(newStatus)
+        }
+    }
+    onInitializationStatusChanged?.invoke(initializationStatus)
```

Comment:
For consistency I would make use of `updateStatus` here:
```suggestion
    updateStatus(TableTopSceneViewStatus.Initializing)
```